src/styles/card-styles.css

```
@@ -0,0 +1,98 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --text-dark-color: #060b35;
+  --text-light-color: #616070;
+  --button-blue-color: #00acdc;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__description {
+  margin: 0 auto 4px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--text-dark-color);
+}
+
+.card__product-cod {
+  margin-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-light-color);
+}
+
+.card__review {
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__review-rate {
+  font-size: 10px;
+  font-weight: 400;
+
+  line-height: 14px;
+  color: var(--text-dark-color);
+}
+
+.card__price-item {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-name {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-light-color);
+}
+
+.card__price-sum {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--text-dark-color);
+}
+
+.card__button_buy {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+
+  border: 1px solid var(--button-blue-color);
+  background: var(--button-blue-color);
+  border-radius: 5px;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  transition: 0.5s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```